src/Forms/GridField/FormAction/SessionStore.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace SilverStripe\Forms\GridField\FormAction;
+
+use SilverStripe\Control\HTTPRequest;
+
+/**
+ * Stores GridField action state in the session in exactly the same way it has in the past
+ */
+class SessionStore extends AbstractRequestAwareStore implements StateStore
```

Comment:
`AbstractRequestAwareStore` is implementing `StateStore` now. This implements statement is kind of redundant.